src/components/notebook-utils.js

```diff
@@ -48,8 +48,18 @@ export const notebookNameValidator = existing => ({
     message: 'already exists'
   }
 })
+// removes leading dirs and a file ext suffix on paths
+export const getDisplayName = name => stripExtension(getFileName(name))
```

Comment:
Since `stripExtension` and `getFileName` aren't used anywhere else, you should remove them.
```suggestion
export const getDisplayName = str => /([^/]+?)(?:\.[^.]+)?$/.exec(str)[1]
```

Generally, we'd avoid a regex, since it's complicated to parse for other developers; I'm suggesting it here only because it requires trimming before and after the string.

Reply:
`getFileName` is used in `Analysis.js` (probably missed a ctrl+f because it's collapsed by default by github). I like the file utilities being separate too to support a possible refactor I discuss here https://github.com/DataBiosphere/terra-ui/pull/2507/files#r637038725

Reply:
In that case, here's my suggestion:
```suggestion
export const getDisplayName = _.flow(stripExtension, getFileName)
```